plugins/chat/assets/javascripts/discourse/controllers/preferences-chat.js

```diff
@@ -13,18 +13,32 @@ const CHAT_ATTRS = [
   "ignore_channel_wide_mention",
   "chat_sound",
   "chat_email_frequency",
+  "chat_header_indicator_preference",
 ];
 
 const EMAIL_FREQUENCY_OPTIONS = [
   { name: I18n.t(`chat.email_frequency.never`), value: "never" },
   { name: I18n.t(`chat.email_frequency.when_away`), value: "when_away" },
 ];
 
+const HEADER_INDICATOR_OPTIONS = [
```

Comment:
let's extract some constants here for  never/all_new/dm_and_mentions and use them everywhere